build-scripts/ocp-artifacts/publish-oc-binary.sh

```diff
@@ -46,7 +46,7 @@ pkg_tar() {
 
 OSE_VERSION=$1
 VERSION=$2
-RPM_PATH=/mnt/rcm-guest/puddles/RHAOS/plashets/${OSE_VERSION}/building/%s/os/Packages
+RPM_PATH=/mnt/data/pub/RHOCP/plashets/${OSE_VERSION}/building/%s/os/Packages
```

Comment:
Although this is not strictly required, it would be nice if we follow the same hierarchy like:
```
https://ocp-artifacts.hosts.prod.psi.rdu2.redhat.com/pub/RHOCP/plashets/4.10/stream/el7/2023-03/202303221742/
https://ocp-artifacts.hosts.prod.psi.rdu2.redhat.com/pub/RHOCP/plashets/4.10/stream/el7/latest
```

Reply:
sounds good to me, maybe subject for a new PR though since this is just addressing the migration to ocp-artifacts